internal/service/s3/bucket_replication_configuration.go

```diff
@@ -38,6 +38,11 @@ func ResourceBucketReplicationConfiguration() *schema.Resource {
 				Required:     true,
 				ValidateFunc: verify.ValidARN,
 			},
+			"token": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringLenBetween(0, 1024),
```

Comment:
just curious, is the max length documented in the AWS API reference?

Reply:
No, I just pulled the value out of thin air -- my next iteration steals the existing impl. from the object lock config, so this will vanish :)